packages/lodestar/src/chain/beaconProposerCache.ts

```diff
@@ -0,0 +1,37 @@
+import {MapDef} from "../util/map";
+import {Epoch} from "@chainsafe/lodestar-types";
+import {IMetrics} from "../metrics";
+
+const PROPOSER_PRESERVE_EPOCHS = 2;
+export type ProposerPreparationData = {
+  validatorIndex: string;
```

Comment:
why is `validatorIndex` is typed as string? we have our own `ValidatorIndex` type

Reply:
yea i was using that earlier, but Lion suggested  to avoid to/fro conversion between validator<>beacon for beacon proposer data  as this is mostly to populate a lookup index at the BN.

Reply:
👍 so maybe just drop a comment here